### PR TITLE
Provisioning: sync samples

### PR DESCRIPTION
**Pull request by kornerc**
**What is this feature?**

Some provisioning examples are outdated, sync them with the docs.
Follow up to #80894.

These were the files used for syncing:
- `conf/provisioning/access-control/sample.yaml`: `docs/sources/administration/roles-and-permissions/access-control/rbac-grafana-provisioning`
- `conf/provisioning/alerting/sample.yaml`: `docs/sources/alerting/set-up/provision-alerting-resources/file-provisioning/index.md`
- `conf/provisioning/dashboards/sample.yaml`: `docs/sources/administration/provisioning/index.md`
- `conf/provisioning/notifiers/sample.yaml`: `docs/sources/administration/provisioning/index.md`
- `conf/provisioning/plugins/sample.yaml`: `docs/sources/administration/provisioning/index.md`

Furthermore, I tried to fix some inconsistencies in the provisioning docs examples (but I can revert the commit if this should not be part of this commit)

**Why do we need this feature?**

...

**Who is this feature for?**
...

**Which issue(s) does this PR fix?**:
...

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: conf/provisioning/access-control/sample.yaml
Comment (gamab) [inline conf/provisioning/access-control/sample.yaml:1]: The changes made to scopes in this file should be undone, `users:read` applies to `global.users:*`

**Review by gamab — DISMISSED** [review]

File: conf/provisioning/access-control/sample.yaml
Comment (kornerc) [reply conf/provisioning/access-control/sample.yaml:1]: I've reverted these changes.

Should I also change that in the current docs?
https://github.com/grafana/grafana/blob/aaada8efcf2a99131050b715a172b5aba3f85a8b/docs/sources/administration/roles-and-permissions/access-control/rbac-grafana-provisioning/index.md?plain=1#L82

Comment (gamab) [reply conf/provisioning/access-control/sample.yaml:1]: Thanks!

> Should I also change that in the current docs?

Yes if you could that'd be great :)

File: docs/sources/administration/roles-and-permissions/access-control/rbac-grafana-provisioning/index.md
Comment (kornerc) [inline docs/sources/administration/roles-and-permissions/access-control/rbac-grafana-provisioning/index.md:1]: @gamab please have a look if I changed scopes correctly

File: conf/provisioning/access-control/sample.yaml
Comment (kornerc) [reply conf/provisioning/access-control/sample.yaml:1]: Done, see other comment

**Review by brendamuir — APPROVED** [review]
for alerting docs-lgtm

**Comment by kornerc** [comment]
Should I split this PR into smaller PRs to make the review and approval process easier?

**Review by marefr — APPROVED** [review]
Plugins provisioning LGTM

**Review by gamab — APPROVED** [review]
Access Control part :heavy_check_mark:

**Comment by kornerc** [comment]
I will close this PR and split it into smaller PRs.
This should simplify the review because less reviewers will be needed per PR.

**Pull request closed without merging**